### PR TITLE
fix #1690: unique ACMG map for all causatives

### LINKED ISSUE #1690: ACMG classification not appearing in the causatives page, in some cases
bild

these CPT2 variants in case 2007 are both classified with ACMG but only for one of them the classification appears.

This appen for several variants, I can see that if there is a "-" the classification has not been done but if the field after "ACMG:" is empty the classification is done but not appearing. See the difference in the pic below

bild

### PR DESCRIPTION
**Pull request by northwestwitch**
fix #1690

**How to test**:
1. Install this branch on scout stage and go to causatives page of one institute
1. Make sure that each causative has an acmg classification missing (-) or a value (P, LP, VUS, LB, B)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Test before (from master branch):
image


Test after (after pulling this branch):
image

Works!

**Comment by northwestwitch** [comment]
This is tested and ready for review!

File: scout/server/blueprints/cases/views.py [hunk 1]
```diff
@@ -517,15 +517,13 @@ def causatives(institute_id):
...
-            if isinstance(variant_obj.get("acmg_classification"), int):
-                acmg_code = ACMG_MAP[variant_obj["acmg_classification"]]
-                variant_obj["acmg_classification"] = ACMG_COMPLETE_MAP[acmg_code]
-
             all_variants[variant_obj["variant_id"]] = []
+
         all_variants[variant_obj["variant_id"]].append((case_obj, variant_obj))
 
-    return dict(institute=institute_obj, variant_groups=all_variants)
+    acmg_map = {key: ACMG_COMPLETE_MAP[value] for key, value in ACMG_MAP.items()}
```
Comment (dnil) [inline scout/server/blueprints/cases/views.py]: 👍

Comment (northwestwitch) [reply scout/server/blueprints/cases/views.py]: Thanks!

**Comment by northwestwitch** [comment]
> Nicely caught!

The process was painful though ;)

**Pull request merged**